src/annotator/sidebar.coffee

```diff
@@ -25,7 +25,13 @@ module.exports = class Sidebar extends Host
   gestureState: null
 
   constructor: (element, config) ->
+    if config.theme == 'clean' || config.externalContainerSelector
+      delete config.pluginClasses.BucketBar
```

Comment:
I think it is surprising to have the constructor mutate the `config.pluginClasses` object here. I would have preferred to instead create a modified copy. There is also potentially a problem if anyone were ever to subclass `Sidebar` in future - although I would suggest that would almost certainly be a bad idea anyway.